domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/Child.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.cmc.domain.models.statementofmeans;
+
+import lombok.Builder;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.util.Objects;
+import javax.validation.constraints.NotNull;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@Builder
+public class Child {
+
+    public enum AgeGroupType {
+        UNDER_11,
+        BETWEEN_11_AND_15,
+        BETWEEN_16_AND_19;
+    }
+
+    @NotNull
+    private final AgeGroupType ageGroupType;
+
+    @NotNull
+    private final Integer howMany;
```

Comment:
I do not like fields which some like a question. What about renaming to `numberOfChildreen` or `numberOfChildreenInGroup`